components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -141,32 +151,38 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
         setTimeout(() => testSiteURL(onSuccess, onError, location.origin), 1000);
     }, []);
 
-    const dataKey = Object.keys(data);
+    const accData: AccordionDataType[] = [];
+    const dataKeys = Object.keys(data);
+
+    dataKeys.forEach((key: string) => {
```

Comment:
Another option is something like this:
```js
Object.entries(data).map(([accordionKey, accordionData]) => {
    console.log('accordionKey', accordionKey);
    console.log('accordionData', accordionData);
});
```

Reply:
@mkraft is right ... the `entries` method might be the better option here